.gitignore

```diff
@@ -3,3 +3,4 @@
 .vercel
 node_modules
 yarn-error.log
+.vscode
```

Comment:
I think we have workspace settings here that could be useful to others? I wouldn't ignore it

Reply:
I thought it was default to exclude it, but after reading there seem to be good arguments for it.
The debugging settings are in there, so if anyone gets it to work with VS Code it's shared directly.